PhysicsTools/NanoAOD/python/photons_cff.py

```diff
@@ -207,12 +212,24 @@ def make_bitmapVID_docstring(id_modules_working_points_pset):
         mvaID_Fall17V1p1 = Var("userFloat('mvaID_Fall17V1p1')",float,doc="MVA ID score, Fall17V1p1",precision=10),
         mvaID_WP90 = Var("userInt('mvaID_WP90')",bool,doc="MVA ID WP90, Fall17V2"),
         mvaID_WP80 = Var("userInt('mvaID_WP80')",bool,doc="MVA ID WP80, Fall17V2"),
+        pfPhoIso03 = Var("photonIso()",float,doc="PF absolute isolation dR=0.3, photon component (uncorrected)"),
+        trkSumPtHollowConeDR03 = Var("trkSumPtHollowConeDR03()",float,doc="Sum of track pT in a hollow cone of outer radius, inner radius"),
```

Comment:
1. I think we will use this for Run 3 as well, it's one of the variables we cut on during the diphoton preselection.
2. Sure, what precision should I set?
3. @simonepigazzini, @rchatter

Reply:
@lfinco Do you think we can switch safely to the offline (reco) isolation definition and drop the HLT-like ones?

Reply:
Cuts at HLT are at 30 and 22 GeV. 
If with offline reco isolation you mean the pf photon isolation, the two might be quite different.
Given that with the preselection we want to be as close as possible to the trigger selection, using trkSumPtHollowConeDR03 would be in principle better. The usage of pfPhoIso03 instead might have an impact on the trigger efficiencies (which are computed on top of the preselection) and this impact should be evaluated.

Reply:
Track isolation at HLT should be close to charged hadron isolation offline. It would be useful to compare the two distributions (track iso offline and charged hadron iso), to see how close they are.